### ISSUE TITLE
Check efficacy of a posteriori training

### ISSUE THREAD

**Issue by SCiarella** [DEEPDIP-project/CoupledNODE.jl#66]
We have to confirm that the a posteriori training is effectively going in the right direction. The loss does not decrease enough to confirm this and it is very unstable suggesting some problems.
A basic test could be
- [ ] confirm that a posteriori training converges to something better than no closure

then later we have to confirm that
- [ ] given a specific CNN architecture and data structure, CoupleNODE produces results comparable to NeuralClosure

**Comment by SCiarella** [comment]
- [ ] implement a callback function that plots some 'validation loss' measured on a constant dataset (not used for training)
